server/rooms.ts

```diff
@@ -56,7 +57,9 @@ interface ChatRoomTable {
 	title: string;
 	desc: string;
 	userCount: number;
+	section?: RoomSection;
 	subRooms?: string[];
+	prizewinner?: true;
```

Comment:
After sleeping on it, I think `spotlight` is a better term than `prizewinner`.

Specifically, `spotlight?: string`, so it can be `spotlight: "PSPL Winner",` but would make it easier for us to make spotlights with different names in the future. Perhaps also rename the commands to be similarly generic, but that part's less important than the protocol.

Reply:
I'll change to spotlight, but the generalizing `/psplwinnerroom` etc should probably be in a different PR

Reply:
Sure.